pkg/controllers/qat/controller.go

```diff
@@ -78,13 +78,15 @@ func (c *controller) GetTotalObjectCount(ctx context.Context, clnt client.Client
 func (c *controller) NewDaemonSet(rawObj client.Object) *apps.DaemonSet {
 	devicePlugin := rawObj.(*devicepluginv1.QatDevicePlugin)
 	yes := true
+	pluginAnnotations := devicePlugin.ObjectMeta.DeepCopy().Annotations
```

Comment:
`UpdateDaemonSet` needs changing too: if the user edits existing `QatDevicePlugin` the annotations are not added/removed

Reply:
done